test/integration/render/run_render_tests.ts

```diff
@@ -10,10 +10,10 @@ import http from 'http';
 import puppeteer, {Page, Browser} from 'puppeteer';
 import v8toIstanbul from 'v8-to-istanbul';
 import {localizeURLs} from '../lib/localize-urls';
-import type {default as MapLibreGL, Map, CanvasSource, PointLike, StyleSpecification} from '../../../dist/maplibre-gl';
+import type {Map, CanvasSource, PointLike, StyleSpecification} from '../../../dist/maplibre-gl';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
-let maplibregl: typeof MapLibreGL;
+let maplibregl: any;
```

Comment:
```suggestion
let maplibregl: typeof maplibreglModule;
```

this would probably be more type-safe in tests :) (with this fixes it handles the problem you had with RTL - the return type of `getRTLTextPluginStatus` was `void` instead of a string)